fix parsing for "game lose 2,2;0,2" / count timeouts separately from ties

gyorokpeter opened the pull request.

reddyalready commented:
Sorry for this being open a while, just wanted to get the v10 stuff from last year in, so we didn't have any conflicts. In light of that, is this still good to go?

@joffat, any thoughts given you've worked on timeouts most recently?

reddyalready commented:
Also, I hope it's ok to change the winner bit to "Veggie dinner". We've historically wanted to keep the winner winner bit as we started this whole thing in part because PUBG was so popular and it reminds us of that. I definitely agree though that we need to be more responsible about our message with regards to the environment, hence veggie.

gyorokpeter commented:
I'm fine with the victory message.
I originally added both an overall timeout counter and changed the individual game result to show "time" instead of "tied". After the rebase, only the latter change remains (although I also removed the extra "Details" heading).
